src/routers/user.ts

```diff
@@ -94,22 +117,50 @@ user.get(
   '/users',
   userQuerySubstringLimitValidator,
   methodLoggerMiddleware,
-  async (req: ValidatedRequest<QuerySubstringLimitSchema>, res: Response, next: NextFunction) => {
+  checkTokenMiddleware,
+  async (
+    req: ValidatedRequest<QuerySubstringLimitSchema>,
+    res: Response,
+    next: NextFunction
+  ) => {
     try {
       const { loginsubstring } = req.query;
       const { limit } = req.query;
 
       const result = await userService.getUsers(loginsubstring, limit);
       res.status(200).json({ users: result });
     } catch (e) {
-      next(e)
+      next(e);
     }
   }
 );
 
-user.get('/error', methodLoggerMiddleware, function() {
-  //throw new Error("I AM UNHANDLED EXEPTION>>>>>");
-  //new Promise((res,rej)=>{rej('I AM UNHANDLED PROMISE REJECTION>>>>>')});
-});
+user.post(
+  '/user/authenticate',
```

Comment:
Iirc in the task method should be `login`

Reply:
fixed